### PR TITLE
Few Changes 

### PR DESCRIPTION
**Pull request by niravpatel006**

### CODE REVIEW

File: methods.tex [hunk 1]
```diff
@@ -308,7 +318,7 @@ \subsection{Data Communication Protocol}
 often align with the structure of most databases, allowing for easy translation
 between disk and memory.
 
-Agent-based fuel cycle modeling, however, subsumes physical modeling frameworks.
+Agent-based fuel cycle modeling, however, subsumes physical modeling frame-works.
```
Comment (scopatz) [inline methods.tex]: frameworks is one word, please remove hyphen

File: paper.tex [hunk 2]
```diff
@@ -90,8 +90,8 @@
 science and software development aspects of archetype development
 that can be addressed algorithmically, allowing the developer to focus on 
 modeling the physics, social policies, and economics they want to simulate. \cycpp passes through 
-the code three times to perform the following tasks: normalizing the code via 
-the C preprocessor, accumulation of notations, and code generation. Not only
+the code three times to perform the following tasks: I.)Normalizing the code via 
+the C preprocessor,II.)Accumulation of notations, and III.)Code generation. Not only
```
Comment (scopatz) [inline paper.tex]: please remove numbers and revert to lowercase.

File: conclusion.tex [hunk 3]
```diff
@@ -15,7 +15,7 @@ \section{Conclusions}
 algorithmically. This moves effort away from humans, who are pursing physics and
 economics, and onto computers. This automation happens by default for
 archetype development and reduces manual code writing by approximately
-10 times.
+10 times or may be more but not less.
```
Comment (scopatz) [inline conclusion.tex]: The new text says the same thing as approximately. Please remove

**Comment by scopatz** [comment]
Thanks @niravpatel006! Please add yourself to the license file as well.

**Comment by scopatz** [comment]
whoops I meant authors file.

File: intro.tex [hunk 4]
```diff
@@ -1,128 +1,143 @@
...
-is the high degree of agent specialization. Standard agent-based simulators
-are characterized as having a few types of agents, often only one and 
-almost always less than five \cite{taylor2014agent}. The agent types are then 
-specialized 
-when they are instantiated \emph{in situ}. This model is insufficient for 
-nuclear fuel cycle applications.  For example, it would be unwise to have a single facility model 
+What sets nuclear fuel cycle simulation apart from traditional agent-based simulations
+is the high degree of agent specialization? Standard agent-based simulators
+are characterized as having a few types of agents, often only one and
+almost always ($<5) less than five \cite{taylor2014agent}. The agent types are then
```
Comment (scopatz) [inline intro.tex]: Maybe just remove the piece in parentheses. It is said in words already

File: methods.tex [hunk 5]
```diff
@@ -114,11 +121,14 @@ \subsection{Automatic Model Templating}
 \label{opt-api}
 \begin{tabular}[p]{|lp{5.25in}|}
 \hline
-\textbf{Function} & \textbf{Description} \\
+\textbf{Function|} & \textbf{Description} \\
```
Comment (scopatz) [inline methods.tex]: Should not have `|` here

File: paper.tex [hunk 6]
```diff
@@ -91,7 +91,7 @@
 that can be addressed algorithmically, allowing the developer to focus on 
 modeling the physics, social policies, and economics they want to simulate. \cycpp passes through 
 the code three times to perform the following tasks: normalizing the code via 
-the C preprocessor, accumulation of notations, and code generation. Not only
+the C preprocessor,accumulation of notations, and code generation. Not only
```
Comment (scopatz) [inline paper.tex]: need space after comma

File: intro.tex [hunk 7]
```diff
@@ -1,128 +1,143 @@
...
 
-What sets nuclear fuel cycle simulation apart from traditional agent-based simulations 
-is the high degree of agent specialization. Standard agent-based simulators
-are characterized as having a few types of agents, often only one and 
-almost always less than five \cite{taylor2014agent}. The agent types are then 
-specialized 
-when they are instantiated \emph{in situ}. This model is insufficient for 
-nuclear fuel cycle applications.  For example, it would be unwise to have a single facility model 
+What sets nuclear fuel cycle simulation apart from traditional agent-based simulations
+is the high degree of agent specialization? Standard agent-based simulators
```
Comment (gonuke) [inline intro.tex]: This is not a question

File: intro.tex [hunk 8]
```diff
@@ -1,128 +1,143 @@
...
+reductions down to a couple of minutes. This has has been accomplished
+without sacrificing an iota of simulation fidelity. Since such
 problems proliferate among agent-based simulators, the methods described here apply beyond \cyclus.
-Though such methods sometimes dive into 
+Though such methods sometimes dive into
 computer science and agent-based modeling details, they are always implemented
-with the express goal of making fuel cycle simulation 
-simpler, easier, faster, more expressive, and validated.
+with the express goal of making fuel cycle simulation
+simpler, easier, faster, more expressive and validated.
```
Comment (gonuke) [inline intro.tex]: I would retain the Oxford comma here after expressive

File: intro.tex (hunk 7 above)
Comment (gonuke) [inline intro.tex]: This is not a question. (even though it starts with "What")

**Pull request merged**